src/cpp/desktop/DesktopGwtCallback.cpp

```diff
@@ -244,6 +255,24 @@ QString GwtCallback::getExistingDirectory(const QString& caption,
    return createAliasedPath(result);
 }
 
+void GwtCallback::onClipboardChanged(QClipboard::Mode mode)
+{
+    // if this is a change in the selection contents, track it
+    if (mode == QClipboard::Selection)
+    {
+        QClipboard* cb = QApplication::clipboard();
+        QString text = cb->text(mode);
+
+        // when one clicks on an Ace instance, a hidden length-one selection
+        // will sneak in here. explicitly screen those out.
+        if (text == QString::fromUtf8("\x01"))
+            return;
```

Comment:
Do we know *why* Ace does this? I'd leave it unless we know it's a bug or is causing downstream trouble for us (perhaps they're using it as a sentinel of some kind?).